shared/src/metabase/shared/util/currency.cljc

```diff
@@ -0,0 +1,851 @@
+          :code "RSD",
+          :name_plural "Serbian dinars"}],
+   [:RUB {:symbol "₽",
+          :name "Russian Ruble",
+          :symbol_native "₽",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "RUB",
+          :name_plural "Russian rubles"}],
+   [:RWF {:symbol "RWF",
+          :name "Rwandan Franc",
+          :symbol_native "FR",
+          :decimal_digits 0,
+          :rounding 0,
+          :code "RWF",
+          :name_plural "Rwandan francs"}],
+   [:SAR {:symbol "SR",
+          :name "Saudi Riyal",
+          :symbol_native "ر.س.‏",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "SAR",
+          :name_plural "Saudi riyals"}],
+   [:SDG {:symbol "SDG",
+          :name "Sudanese Pound",
+          :symbol_native "SDG",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "SDG",
+          :name_plural "Sudanese pounds"}],
+   [:SEK {:symbol "Skr",
+          :name "Swedish Krona",
+          :symbol_native "kr",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "SEK",
+          :name_plural "Swedish kronor"}],
+   [:SGD {:symbol "S$",
+          :name "Singapore Dollar",
+          :symbol_native "$",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "SGD",
+          :name_plural "Singapore dollars"}],
+   [:SOS {:symbol "Ssh",
+          :name "Somali Shilling",
+          :symbol_native "Ssh",
+          :decimal_digits 0,
+          :rounding 0,
+          :code "SOS",
+          :name_plural "Somali shillings"}],
+   [:SYP {:symbol "SY£",
+          :name "Syrian Pound",
+          :symbol_native "ل.س.‏",
+          :decimal_digits 0,
+          :rounding 0,
+          :code "SYP",
+          :name_plural "Syrian pounds"}],
+   [:THB {:symbol "฿",
+          :name "Thai Baht",
+          :symbol_native "฿",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "THB",
+          :name_plural "Thai baht"}],
+   [:TND {:symbol "DT",
+          :name "Tunisian Dinar",
+          :symbol_native "د.ت.‏",
+          :decimal_digits 3,
+          :rounding 0,
+          :code "TND",
+          :name_plural "Tunisian dinars"}],
+   [:TOP {:symbol "T$",
+          :name "Tongan Paʻanga",
+          :symbol_native "T$",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "TOP",
+          :name_plural "Tongan paʻanga"}],
+   [:TRY {:symbol "TL",
+          :name "Turkish Lira",
+          :symbol_native "TL",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "TRY",
+          :name_plural "Turkish Lira"}],
+   [:TTD {:symbol "TT$",
+          :name "Trinidad and Tobago Dollar",
+          :symbol_native "$",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "TTD",
+          :name_plural "Trinidad and Tobago dollars"}],
+   [:TWD {:symbol "NT$",
+          :name "New Taiwan Dollar",
+          :symbol_native "NT$",
+          :decimal_digits 2,
```

Comment:
new taiwan dollar doesn't have decimals in common usage, it's like 30 ntd to 1 usd